app/cdap/components/common/BoxContainer/index.tsx

```diff
@@ -14,19 +14,25 @@
  * the License.
  */
 
-import { makeStyles } from '@material-ui/core';
+import styled from 'styled-components';
+import { Box } from '@material-ui/core';
 
-export const useStyles = makeStyles({
-  loadingContainer: {
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
-    height: '100%',
-    opacity: 0.5,
-    background: 'white',
-    position: 'absolute',
-    top: 0,
-    width: '100%',
-    zIndex: 2000,
-  },
-});
+export const DirectiveBox = styled(Box)`
```

Comment:
can we move this component to DirectiveInput/Components plz
these might not work as common components
plz move them to specific feature folder

Reply:
Done